modLogger/src/org/aion/log/AionLoggerFactory.java

```diff
@@ -70,12 +75,67 @@
         }
     }
 
+    /** Change INITIALIZE signature to include LOGFILE and LOGPATH */
     public static void init(final Map<String, String> _logModules) {
+        init(_logModules, false, "log");
+    }
+
+    public static void init(final Map<String, String> _logModules, boolean _logToFile, String _logToPath) {
 
         logModules = _logModules;
+        logFile = _logToFile;
+        logPath = _logToPath;
 
         loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
 
+        /** Toggles file appending configurations */
```

Comment:
could you switch all the comments that use the java doc notation `/**` to the comment notation `/*` throughout all the files of the PR

Reply:
Changed the comment notation and added the license headers to relevant files